client/src/app/tabs/bpmn/diagram.bpmn

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" id="Definitions_{{ ID }}" targetNamespace="http://bpmn.io/schema/bpmn" xmlns:modeler="http://camunda.org/schema/modeler/1.0" modeler:executionPlatform="Camunda Platform" modeler:executionPlatformVersion="7.14.0">
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" id="Definitions_{{ ID }}" targetNamespace="http://bpmn.io/schema/bpmn" xmlns:modeler="http://camunda.org/schema/modeler/1.0" modeler:executionPlatform="Camunda Platform" modeler:executionPlatformVersion="7.15.x">
```

Comment:
This is not a semantic version I believe and thus may bite us, as we introduce linting (some day, maybe).

If we keep this a actual semantic version we could use existing utilities (i.e. semver ranges) some day in the future.

Not sure how much this matters right now. I'd not add an `x` without need though.

Reply:
Could harm us in the future for sure, because it will make things harder to check... but maybe it's a matter of when we start working on the feature versioning.

Reply:
Good point.

This is a technical attribute, so we can do it "correctly" right from the beginning. Hence I would propose to use `^7.15.0` WDYT?

Reply:
What would be the benefit over stating the correct stack release version the modeler is compatible with? 

You'd do the matching using actual semver ranges later on (i.e. define the ranges per rule), we do not need to define a range here. Simply state the fact what this diagram was built for (the currently stable release) and we are free to do whatever later on.

Reply:
Agreed